app/assets/src/components/Samples.jsx

```diff
@@ -987,32 +1010,12 @@ class Samples extends React.Component {
       </div>
     );
 
-    const compare_button_inner = (
-      <span>
-        <span
-          className="img-container compare-container"
-          dangerouslySetInnerHTML={{
-            __html: IconComponent.compare(colors.disabledGray)
-          }}
-        />
-        <span>Compare</span>
-      </span>
-    );
-
     let compare_button = (
-      <div className="compare-area">
-        <div className="white">
-          {this.state.selectedSampleIds.length > 0 ? (
-            <a onClick={this.compareSamples} className="compare center">
-              {compare_button_inner}
-            </a>
-          ) : (
-            <a className="compare center btn-disabled">
-              {compare_button_inner}
-            </a>
-          )}
-        </div>
-      </div>
+      <CohortButton
+        label="Compare"
+        action={this.compareSamples}
```

Comment:
action -> onClick
We should keep descriptive names and in this case, jut inherit the name of the underlying element.

Reply:
ok